<!doctype html>
    <html lang="en">
    <head>
      <meta charset="utf-8" />
      <link rel="canonical" href="/topics/neuroscience/sanger-sequencing">
      <link rel="preload" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/3/fonts/nexus/1.0/serif/NexusSerifWebPro-Regular.woff" as="font" type="font/woff" crossorigin="anonymous">
      <link rel="preload" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/3/fonts/nexus/1.0/sans/NexusSansWebPro-Regular.woff" as="font" type="font/woff" crossorigin="anonymous">
      <link rel="preload" href="/assets/style.css" as="style">
      <link rel="preload" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/3/fonts/nexus/1.0/sans/NexusSansWebPro-Bold.woff" as="font" type="font/woff" crossorigin="anonymous">
      <link rel="preload" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/3/fonts/nexus/1.0/sans/NexusSansWebPro-Italic.woff" as="font" type="font/woff" crossorigin="anonymous">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <title>Sanger Sequencing - Methods beta | ScienceDirect Topics</title><meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR"><link rel="canonical" href="https://www.qa.sciencedirect.com/topics/neuroscience/sanger-sequencing" /><meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel='shortcut icon' href='https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico' type='image/x-icon' />
      <link rel='icon' href='https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico' type='image/x-icon'>
      
      <script>
        (function(h,o,t,j,a,r){
          h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
          h._hjSettings={hjid:1350649,hjsv:6};
          a=o.getElementsByTagName('head')[0];
          r=o.createElement('script');r.async=1;
          r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
          a.appendChild(r);
        })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
      </script>
      <!-- begin mPulse embed code -->
      <script crossorigin="anonymous">
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            js.crossorigin = "anonymous";
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
      <script id="adobe-dtm" src=https://assets.adobedtm.com/376c5346e33126fdb6b2dbac81e307cbacfd7935/satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c-staging.js></script>
    </head>
    <body>
      <a class="skip" title="Skip to Main content" href="#main_content">
        Skip to Main content
      </a>
      <div id="react-header-root" class="react-header-root"><header id="gh-cnt" data-reactroot=""><div id="gh-main-cnt" class="u-flex-center-ver u-position-relative u-padding-s-hor u-padding-m-hor-from-sm u-padding-l-hor-from-lg"><a id="gh-branding" class="u-flex-center-ver" href="/" aria-label="Science Direct home page" data-aa-region="header" data-aa-name="ScienceDirect"><img class="gh-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/24/images/elsevier-non-solus-new-grey.svg" alt="Elsevier logo" height="48" width="54"/><svg xmlns="http://www.w3.org/2000/svg" role="img" version="1.1" height="30" width="138" viewBox="0 0 138 30" class="gh-wordmark u-margin-s-left" fill="#f36d21" aria-labelledby="gh-wm-science-direct" focusable="false" aria-hidden="true" alt="ScienceDirect Wordmark"><title id="gh-wm-science-direct">ScienceDirect</title><g><path class="a" d="M4.23,21a9.79,9.79,0,0,1-4.06-.83l.29-2.08a7.17,7.17,0,0,0,3.72,1.09c2.13,0,3-1.22,3-2.39C7.22,13.85.3,13.43.3,9c0-2.37,1.56-4.29,5.2-4.29a9.12,9.12,0,0,1,3.77.75l-.1,2.08a7.58,7.58,0,0,0-3.67-1c-2.24,0-2.91,1.22-2.91,2.39,0,3,6.92,3.61,6.92,7.8C9.5,19.1,7.58,21,4.23,21Z"></path><path class="a" d="M20.66,20A6.83,6.83,0,0,1,16.76,21c-3,0-5.23-2.18-5.23-6.29,0-4.29,2.91-6.11,5.28-6.11,2.16,0,3.67.55,3.85,2.11,0,.23,0,.57,0,.86H18.81c0-1-.55-1.25-1.9-1.25a2.87,2.87,0,0,0-1.35.21c-.21.13-1.85.94-1.85,4.11s1.9,4.65,3.59,4.65a5.91,5.91,0,0,0,3.2-1.2Z"></path><path class="a" d="M23.75,6.9a1.45,1.45,0,0,1-1.3-1.46,1.32,1.32,0,1,1,2.63,0A1.5,1.5,0,0,1,23.75,6.9ZM22.76,9h2V20.74h-2Z"></path><path class="a" d="M29.55,14.6V15c0,2.81,1.38,4.34,3.85,4.34a6.37,6.37,0,0,0,3.69-1.22l.16,1.82A7.94,7.94,0,0,1,32.77,21c-3,0-5.3-2.29-5.3-6.16,0-4.06,2.21-6.24,5.25-6.24,3.61,0,4.73,1.87,4.73,6ZM35.63,13c-.08-2.29-1.09-2.7-3-2.7A3.78,3.78,0,0,0,31,10.7,3.7,3.7,0,0,0,29.76,13Z"></path><path class="a" d="M49.7,20.74h-2s.1-2.73.08-5.1c0,0,0-1.56,0-2.5-.05-1.79-.21-2.7-2-2.7a4.87,4.87,0,0,0-1.64.31,12.11,12.11,0,0,0-1.95,2.08v7.9h-2v-8.5a19.47,19.47,0,0,0-.1-2.34L39.95,9h1.85l.31,1.74a4.71,4.71,0,0,1,3.82-2.05c2.11,0,3.54.68,3.74,3.09.1,1.17.08,2.34.08,3.51C49.75,17.2,49.7,20.74,49.7,20.74Z"></path><path class="a" d="M61.5,20A6.83,6.83,0,0,1,57.6,21c-3,0-5.23-2.18-5.23-6.29,0-4.29,2.91-6.11,5.28-6.11,2.16,0,3.67.55,3.85,2.11,0,.23,0,.57,0,.86H59.66c0-1-.55-1.25-1.9-1.25a2.87,2.87,0,0,0-1.35.21c-.21.13-1.85.94-1.85,4.11s1.9,4.65,3.59,4.65a5.91,5.91,0,0,0,3.2-1.2Z"></path><path class="a" d="M64.75,14.6V15c0,2.81,1.38,4.34,3.85,4.34a6.37,6.37,0,0,0,3.69-1.22l.16,1.82A7.94,7.94,0,0,1,68,21c-3,0-5.3-2.29-5.3-6.16,0-4.06,2.21-6.24,5.25-6.24,3.61,0,4.73,1.87,4.73,6ZM70.84,13c-.08-2.29-1.09-2.7-3-2.7a3.78,3.78,0,0,0-1.56.36A3.7,3.7,0,0,0,65,13Z"></path><path class="a" d="M81.21,20.74H75.83V5h5.62c5.54,0,7.46,4.21,7.46,7.8C88.91,16.26,86.93,20.74,81.21,20.74Zm-.1-14H77.88V19.07h3c4,0,5.75-2.31,5.75-6.24C86.59,10.15,85.34,6.7,81.11,6.7Z"></path><path class="a" d="M92.86,6.9a1.45,1.45,0,0,1-1.3-1.46,1.32,1.32,0,1,1,2.63,0A1.5,1.5,0,0,1,92.86,6.9ZM91.87,9h2V20.74h-2Z"></path><path class="a" d="M104.48,10.83l-1.64.47c0-.18-.08-1-.83-1-1.14,0-2.08,1.9-2.5,2.91v7.49h-2V12.18a18.78,18.78,0,0,0-.1-2.29L97.3,9h1.85l.34,1.87a3.22,3.22,0,0,1,2.68-2.16,2,2,0,0,1,2.26,1.72c0,.18.05.29.05.31Z"></path><path class="a" d="M107.44,14.6V15c0,2.81,1.38,4.34,3.85,4.34A6.37,6.37,0,0,0,115,18.11l.16,1.82A7.94,7.94,0,0,1,110.67,21c-3,0-5.3-2.29-5.3-6.16,0-4.06,2.21-6.24,5.25-6.24,3.61,0,4.73,1.87,4.73,6ZM113.53,13c-.08-2.29-1.09-2.7-3-2.7a3.78,3.78,0,0,0-1.56.36A3.7,3.7,0,0,0,107.65,13Z"></path><path class="a" d="M126.24,20a6.83,6.83,0,0,1-3.9,1.09c-3,0-5.23-2.18-5.23-6.29,0-4.29,2.91-6.11,5.28-6.11,2.16,0,3.67.55,3.85,2.11,0,.23,0,.57,0,.86H124.4c0-1-.55-1.25-1.9-1.25a2.87,2.87,0,0,0-1.35.21c-.21.13-1.85.94-1.85,4.11s1.9,4.65,3.59,4.65a5.91,5.91,0,0,0,3.2-1.2Z"></path><path class="a" d="M134.51,20.45a7.36,7.36,0,0,1-2.7.62c-1.53,0-2.63-.86-2.63-2.94V10.52H127V9h2.13V5.81h2V9h3.09v1.56h-3.09v7c0,1.33.34,1.85,1.25,1.85a5.66,5.66,0,0,0,2-.55Z"></path></g></svg></a><div class="gh-nav-cnt"><div class="gh-nav-links-container gh-nav-links-container-h"><nav aria-label="links" class="gh-nav gh-nav-links gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item gh-move-to-spine"><a class="anchor button-link-primary gh-nav-action" href="/browse/journals-and-books" data-aa-region="header" data-aa-name="Journals &amp; Books"><span class="anchor-text">Journals &amp; Books</span></a></li></ul></nav><nav aria-label="utilities" class="gh-nav gh-nav-utilities gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item gh-search-toggle"><a class="anchor button-link-primary gh-nav-action gh-icon-btn" href="/search/advanced" data-aa-button="search-in-header-opened-from-foundationalcontent" aria-label="Opens ScienceDirect Search"><span class="anchor-text"></span><svg focusable="false" viewBox="0 0 100 128" aria-hidden="true" alt="Search" width="18.75" height="24" class="icon icon-search gh-icon"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></a></li><li class="gh-nav-item gh-move-to-spine gh-help-button"><a class="anchor button-link-primary gh-nav-action gh-icon-btn" href="https://service.elsevier.com/app/answers/detail/a_id/25793/supporthub/sciencedirect/" target="_blank" rel="noopener noreferrer" aria-label="Science Direct help page opens in new window"><span class="anchor-text"></span><svg focusable="false" viewBox="0 0 114 128" aria-hidden="true" alt="ScienceDirect help page" width="21.375" height="24" class="icon icon-help gh-icon"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></a></li></ul></nav></div></div><div class="gh-profile-container gh-move-to-spine"><a class="link-button u-margin-m-left link-button-secondary link-button-small" role="button" href="/user/register?targetURL=%2Ftopics%2Fneuroscience%2Fsanger-sequencing%2Fmethod" id="gh-cta-btn" data-aa-region="header" data-aa-name="Register"><span class="link-button-text">Create account</span></a><a class="link-button u-margin-m-left link-button-primary link-button-small" role="button" href="/user/login?targetURL=%2Ftopics%2Fneuroscience%2Fsanger-sequencing%2Fmethod" id="gh-signin-btn" data-aa-region="header" data-aa-name="Sign in"><span class="link-button-text">Sign in</span></a></div><div id="gh-mobile-menu" class="mobile-menu"><div class="gh-hamburger u-fill-grey7 u-margin-m-left"><button class="button-link u-flex-center-ver button-link-primary" aria-label="Toggle mobile menu" aria-expanded="false" type="button"><svg class="gh-hamburger-svg-el gh-hamburger-closed" role="img" aria-hidden="true" height="18" width="40"><path d="M0 14h40v2H0zm0-7h40v2H0zm0-7h40v2H0z"></path></svg><span class="button-link-text"> </span></button></div><div id="gh-overlay" class="mobile-menu-overlay u-overlay u-display-none" role="button" tabindex="-1"></div><div id="gh-drawer" aria-label="Mobile menu" class="" role="navigation"><header id="gh-drawer-header" class="u-padding-s text-s"><div class="gh-dh-opt"><div><p class="u-margin-s-bottom"></p><div class="gh-dh-actions"><a class="anchor" href="/user/login?targetURL=%2Ftopics%2Fneuroscience%2Fsanger-sequencing%2Fmethod" data-aa-region="header" data-aa-name="Sign in"><span class="anchor-text">Sign in</span></a><a class="anchor u-margin-m-left" href="/user/register?targetURL=%2Ftopics%2Fneuroscience%2Fsanger-sequencing%2Fmethod" data-aa-region="header" data-aa-name="Register"><span class="anchor-text">Create account</span></a></div></div></div></header><div class="gh-nav-cnt"><div class="gh-nav-links-container"><nav aria-label="links" class="gh-nav gh-nav-links gh-nav-v"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item"><a class="anchor button-link-primary gh-nav-action" href="/browse/journals-and-books" data-aa-region="header" data-aa-name="Journals &amp; Books"><span class="anchor-text">Journals &amp; Books</span></a></li></ul></nav><nav aria-label="utilities" class="gh-nav gh-nav-utilities gh-nav-v"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item"><a class="anchor button-link-primary gh-nav-action" href="https://service.elsevier.com/app/answers/detail/a_id/25793/supporthub/sciencedirect/" target="_blank" rel="noopener noreferrer"><span class="anchor-text">Help</span></a></li></ul></nav></div></div><div id="gh-mob-inst-cnt"></div></div></div></div></header></div>
      <link rel="stylesheet" href="/assets/protocols-plus/assets/css/protocols.css">
<div id="react-root" class="react-root">
	<div data-reactroot="">
	  <main id="main_content" role="main" class="notpdfdownload">
		<section class="topic-definition-wrapper">
			<div class="definition-inner padding-resp-m u-dark-theme">
				<div class="topic-title">
					<h2 class="pagetype">Methods<span class="beta">Beta</span></h2>
					<a class="anchor gototopic" href="/topics/neuroscience/sanger-sequencing"><span class="anchor-text">Go to original topic page</span></a>
					<h1 class="mb-f-serif u-text-light alt-xl">
						Sanger Sequencing
					</h1>
				</div>
				<div class="definition-container">
					<section class="topic-definition">
						<p>Sanger sequencing is the process of selective incorporation of chain-terminating dideoxynucleotides by DNA polymerase during in vitro DNA replication; it is the most widely used method for the detection of SNVs. Because both alleles of an autosomal locus are sequenced concurrently and are displayed as an analogue electropherograms, Sanger sequencing is unable to detect mosaic alleles below a threshold of 15–20% (Rohlin et al., 2009) and can miss a significant proportion of low-level mosaic mutations (Jamuar et al., 2014). In addition, mosaic mutations at higher allele fractions are miscalled “germ line,” which highlights the limitations of Sanger sequencing in detecting mosaicism on both ends of the spectrum (Jamuar et al., 2014).
						</p>
						<p class="source">
							From:
							<a href="/science/article/pii/B9780128001059000123" class="anchor definition-source" data-aa-region="aa-tp-definition-book-title" data-aa-name="Genomics, Circuits, and Pathways in Clinical Neuropsychiatry" data-hack="#"><span class="anchor-text">Genomics, Circuits, and Pathways in Clinical Neuropsychiatry, 2016</span></a>
						</p>
					</section>
					<section class="vertical-divider">
					<div class="divider"></div>
				</section>
					<section class="header-movie">
					<h3>Video overview</h3>
					<div class="videoWrapper">
						<iframe width="320" height="240" src="https://www.youtube.com/embed/FvHRio1yyhQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
					</div>
				</section>
				</div>
				<section class="related-terms">
					<h3 class="u-h3">
						Related techniques:
					</h3>
					<ul class="list-tags u-clr-blue size-l" data-aa-region="aa-tp-related-terms">

						<li class="list-tags-item"><a class="anchor related-terms-item" href="/topics/neuroscience/pyrosequencing" data-aa-name="Pyrosequencing"><span class="anchor-text">Pyrosequencing</span></a></li>

						<li class="list-tags-item"><a class="anchor related-terms-item" href="/topics/neuroscience/exon" data-aa-name="Exon"><span class="anchor-text">Exon</span></a></li>

						<li class="list-tags-item"><a class="anchor related-terms-item" href="/topics/neuroscience/indel-mutation" data-aa-name="Indel Mutation"><span class="anchor-text">Indel Mutation</span></a></li>

						<li class="list-tags-item"><a class="anchor related-terms-item" href="/topics/neuroscience/dna-template" data-aa-name="DNA Template"><span class="anchor-text">DNA Template</span></a></li>

					</ul>
				</section>
				<div class="action-buttons u-margin-l-top u-show-from-md">
					<div class="u-display-inline-block">
						<button class="button-link u-show-from-md u-margin-m-right action-buttons button-link-primary" aa-data-region="aa-tp-download-topic-PDF" aa-data-name="" type="button"><svg focusable="false" viewbox="0 0 98 128" width="18.375" height="24" class="icon icon-download"><path d="m77.38 56.18l-6.6-7.06-16.78 17.24v-40.36h-1e1v40.34l-17.72-17.24-7.3 7.08 29.2 29.32 29.2-29.32m10.62 17.82v2e1h-78v-2e1h-1e1v3e1h98v-3e1h-1e1"></path></svg><span class="button-link-text">Download as PDF</span></button>
					</div>
					<div class="u-display-inline-block">
						<button class="button-link u-show-from-md action-buttons button-link-primary" type="button"><svg focusable="false" viewbox="0 0 108 128" width="20.25" height="24" class="icon icon-bell"><path d="m44.76 112h-10.39c1.78 9.18 9.9 16 19.63 16s17.85-6.82 19.63-16h-10.39c-1.49 3.64-5.07 6.21-9.24 6.21s-7.75-2.57-9.24-6.21zm-31.68-18c7.08-7.97 19.68-25.19 21.35-48.16l1.18-16.59c3.2-1.33 9.63-3.45 18.17-3.45 8.53 0 15.71 2.12 18.91 3.46l1.21 16.58c1.68 22.98 14.12 40.2 21.09 48.16zm94.91-1.07l-1.63-1.49c-0.2-0.18-20.48-18.96-22.49-46.33l-1.61-21.91-2.27-1.33c-0.41-0.24-10.96-5.87-26.23-5.87-15.26 0-25.05 5.64-25.46 5.88l-2.26 1.33-1.59 21.9c-1.98 27.35-22.77 46.3-22.77 46.3l-1.68 1.49v11.1h108l-0.01-11.07"></path></svg><span class="button-link-text">Set alert</span></button>
					</div>
				</div>
			</div>
		</section>
		<section id="protocols-section" class="protocols-wrapper">
			<div class="protocols-wrapper-inner  padding-resp-hor-m u-padding-l-ver">
				<div class="protocols-container">
					<div class="protocols-main">
						<div class="protocols-header">
							<h2>Protocols</h2>
						</div>
						<div class="protocols protocols-section">
							<div class="citation">
								<h4 class="u-font-serif mb-f-serif">From: <a class="anchor2 protocol-src-title" href="/science/article/pii/B9780128000144000160#s0015">Chapter Sixteen - Sanger Sequencing of Lyssaviruses</a></h4>
								<div>
									<span class="ref-authors">Lorraine M. McElhinney, Denise A Marston, Richard J Ellis, Conrad M. Freuling, Thomas F. Müller, Anthony R. Fooks</span>
									<span class="ref-source-title anchor-text">Current Laboratory Techniques in Rabies Diagnosis, Research and Prevention</span>
									<span class="ref-source-volume anchor-text">2014 • Pages 159-170</span>
								</div>
							</div>
								<p id="p0105">
		The following describes the protocols employed when using the ABI 3130xl, 16 capillary and ABI3730, 48 capillary Genetic Analyzer instruments. Other machines and reagents are available for Sanger sequencing.</span>
	</p>
	<section id="s0020" class="section-expandable">
		<h3 id="st0035" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.1. Preparing Polymerase Chain Reaction Amplicons for the ABI Genetic Analyzers
		</h3>
		<div class="section-content">
		<p id="p0115">
			After a PCR product is amplified from a lyssavirus-positive sample, it needs to be purified and then sequenced using florescence-based <span id="p162"></span><span>terminator cycle sequencing. The following describes the protocol to be followed when using the ABI Big Dye Terminator ready reaction kit. The kit works by providing a ready mix that contains the enzyme, dNTPs, magnesium chloride, buffer and dye terminators (ddNTPs), all at the appropriate quantities. Purified amplicon DNA and one primer are added to the mix and cycled to obtain dye-terminated products, which can be precipitated and analyzed on the ABI genetic analyzers. </span>
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0120">
						Sequencing must be performed in both the reverse and forward direction (i.e. using 3′ and 5′ directed primers) so consensus (complementary) sequence can be obtained.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0125">
						Sequencing in each direction should be replicated, at least in duplicate, prior to publication of the sequence to reduce the likelihood of errors.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0130">
						If cloned PCR products are sequenced, ensure at least five independent clones are sequenced, and use the consensus sequence for publication to ensure minor variants are not used as the representative sequence.
					</p>
				</dd>
			</dl>
		</p>
		</div>
	</section>
	<section id="s0025" class="section-expandable">
		<h3 id="st0040" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.2. Purifying the Polymerase Chain Reaction Product
		</h3>
		<div class="section-content">
		<p id="p0135">
			<span>After visualising the PCR product by agarose gel electrophoresis<span>, the remaining PCR product (amplicon) should be purified prior to sequencing. A number of commercial kits are available for the purification of PCR products (e.g. QiaQuick PCR Purification kit; Qiagen). If the PCR product band appears weak on the gel, it can be loaded and extracted directly from the gel using a commercially available kit (e.g. MinElute gel extraction kit (Qiagen)), which will enable the product to be eluted in as little as 10</span></span>&nbsp;<span>μL Elution Buffer, thereby increasing the concentration. For large-scale purification (96-well plates), automated purification may be preferable (e.g. Millipure Multiscreen HTS PCR 96-well purification system or Beckman Coulter Agencourt AMPure clean up system). </span>
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0140">
						Ensure fresh tips are used between samples to avoid cross-contamination.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0145">
						Elute purified PCR product in 10–50&nbsp;μL Elution Buffer depending on amount of product and purification kit used.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0150">
						Quantify the DNA concentration using a spectrometer (e.g. the Nanodrop or Polarstar Galaxy plate reader).
					</p>
				</dd>
			</dl>
		</p>
		</div>
	</section>
	<section id="s0030" class="section-expandable">
		<h3 id="st0045" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.3. Big Dye Terminator Cycle Sequencing Reactions
		</h3>
		<div class="section-content">
		<p id="p0155">
			On ice, add the purified amplicon DNA (approximately 50–100&nbsp;ng – usually 2–5&nbsp;<span>μL) to the appropriate wells on a 96-well plate. If plasmid DNA is being used, pre-heat the DNA at 96°C for 1 minute and keep on ice. If many primers are being used for sequencing (forward, reverse, etc.), make a </span><span id="p163"></span>mastermix without the primer and add primers to appropriate wells at this stage. See below as an example for calculating amount of DNA to be added.
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0160">
						PCR products: (PCR product length/100)×2)/DNA conc. (ng/μL).
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0165">
						Plasmids: 100/DNA concentration (ng/μL).
					</p>
				</dd>
			</dl>
		</p>
		<div>
			<p id="p0170">
				Make a mastermix using the volumes shown below. The amount of water can be amended to allow for variation in the amount of DNA product added, ensuring final volume is 20&nbsp;<span>μL. The primer employed will depend on the template sequence to be analyzed (examples given in </span>Table 16.1, Table 16.2).
			</p>
			<div class="tables frame-topbot rowsep-0 colsep-0" id="t0010">
				<div class="captions">
					<span id="cap0015">
					<p id="sp0015">
						<span class="label">Table 16.1</span>. Examples of Primers used for Sequencing PCR Amplicons
					</p>
					</span>
				</div>
				<div class="groups">
					<table>
						<thead>
							<tr class="rowsep-1">
								<th scope="col">Gene</th>
								<th scope="col">Forward Primer</th>
								<th scope="col"></th>
								<th scope="col">Reverse Primer</th>
								<th scope="col"></th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td>Partial Nucleoprotein 1st round</td>
								<td>JW12</td>
								<td>ATGTAACACCYCTACAATG</td>
								<td>JW6UNI</td>
								<td>CARTTVGCRCACATYTTRTG</td>
							</tr>
							<tr>
								<td>Partial Nucleoprotein 2nd round</td>
								<td>JW12</td>
								<td>ATGTAACACCYCTACAATG</td>
								<td>JW10UNI</td>
								<td>GTCATYARWGTRTGRTGYTC</td>
							</tr>
							<tr>
								<td>Glycoprotein</td>
								<td>GP01 (GT1)</td>
								<td>GGAAAGATGGTTCCTCAGG</td>
								<td>GP02 (GT1)</td>
								<td>AACTTGAGTTTGCATGCTCC</td>
							</tr>
							<tr>
								<td>Plasmid pCR II</td>
								<td>T7</td>
								<td>TAATACGACTCACTATAGG</td>
								<td>SP6</td>
								<td>ATTTAGGAGACTATAGAA</td>
							</tr>
						</tbody>
					</table>
				</div>
			</div>
			<div class="tables frame-topbot rowsep-0 colsep-0" id="t0015">
				<div class="captions">
					<span id="cap0020">
					<p id="sp0020">
						<span class="label">Table 16.2</span>. Oligonucleotides used in the Long-Distance PCR and RLM-RACE Strategies for EBLV-1 and -2 Sanger Genome Sequencing
					</p>
					</span>
				</div>
				<div class="groups">
					<table>
						<thead>
							<tr class="rowsep-1">
								<th scope="col">Primer</th>
								<th scope="col">Orien-tation</th>
								<th scope="col">Position EBLV-1</th>
								<th scope="col">Position EBLV-2</th>
								<th scope="col">Sequence (5′-3′)</th>
								<th scope="col">Used for</th>
								<th scope="col">EBLV</th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td>L FOR 1</td>
								<td>M</td>
								<td>10264–10285 6</td>
								<td>10228–10250</td>
								<td>GTDGATCAAGAGGTVCGCCATGC</td>
								<td>L LD primer</td>
								<td>1/2</td>
							</tr>
							<tr>
								<td>L REV 2</td>
								<td>G</td>
								<td>11694–11671</td>
								<td>11658–11635</td>
								<td>CTTGTAAATCAACCTKATCCAGTG</td>
								<td>L LD primer</td>
								<td>1/2</td>
							</tr>
							<tr>
								<td>JW12 long SacI</td>
								<td>M</td>
								<td></td>
								<td>44–78</td>
								<td>GCAGA<u>GAGCTC</u>ATGTAACACCTCTACAATGGATGC</td>
								<td>LD PCR</td>
								<td>2</td>
							</tr>
							<tr>
								<td>GG2 long XbaI</td>
								<td>G</td>
								<td></td>
								<td>4946–4913</td>
								<td>CTAAG<u>TCTAGA</u>GGTATTTGGTCCATTAGACAAGC</td>
								<td>LD PCR</td>
								<td>2</td>
							</tr>
							<tr>
								<td>GG2 FOR</td>
								<td>M</td>
								<td></td>
								<td>4913–4935</td>
								<td>GCTTGTCTAATGGACCAAATACC</td>
								<td>LD PCR</td>
								<td>2</td>
							</tr>
							<tr>
								<td>L REV long dist</td>
								<td>G</td>
								<td></td>
								<td>10626–10605</td>
								<td>TGAGGCCATGAAGTCATTCACC</td>
								<td>LD PCR</td>
								<td>2</td>
							</tr>
							<tr>
								<td>JW12 EBLV LD</td>
								<td>M</td>
								<td>52–77</td>
								<td>52–77</td>
								<td>GAAATGTAACACCCCTACAATGGATG</td>
								<td>LD PCR</td>
								<td>1/2</td>
							</tr>
							<tr>
								<td>EBLV G REV LD</td>
								<td>G</td>
								<td>4584–4562</td>
								<td>4596–4574</td>
								<td>GTGGAWGGGTCAGCCAAGGGATG</td>
								<td>LD PCR</td>
								<td>1/2</td>
							</tr>
							<tr>
								<td>EBLV G FOR LD</td>
								<td>M</td>
								<td>4562–4584</td>
								<td>4574–4596</td>
								<td>CATCCCTTGGCTGACCCWTCCAC</td>
								<td>LD PCR</td>
								<td>1/2</td>
							</tr>
							<tr>
								<td>EBLV1 L REV LD</td>
								<td>G</td>
								<td>11221–11197</td>
								<td></td>
								<td>CTGGAAACCCTCTTACAAGGTCTTG</td>
								<td>LD PCR</td>
								<td>1</td>
							</tr>
							<tr>
								<td>Gen end N E2 REV</td>
								<td>G</td>
								<td></td>
								<td>255–232</td>
								<td>GCATTTATTCCGGACAGAATGGAC</td>
								<td>5′ end RLM-RACE</td>
								<td>2</td>
							</tr>
							<tr>
								<td>Gen end N E1 REV</td>
								<td>G</td>
								<td>254–232</td>
								<td></td>
								<td>CATTCATCCCTGACAAGATAGAC</td>
								<td>5′ end RLM-RACE</td>
								<td>1</td>
							</tr>
							<tr>
								<td>EBLV-1 L FOR 16</td>
								<td>M</td>
								<td>10880–10901</td>
								<td></td>
								<td>CTCTGTTAATGTCAGACTTCTC</td>
								<td>3′ end RACE RT</td>
								<td>1</td>
							</tr>
							<tr>
								<td>EBLV-2 L gen end For 4</td>
								<td>M</td>
								<td></td>
								<td>11417–11438</td>
								<td>GACACTTCAATATCTGCTGCAG</td>
								<td>3′ end RACE RT</td>
								<td>2</td>
							</tr>
							<tr>
								<td>EBLV-1 Genomic end L For</td>
								<td>M</td>
								<td>11555–11576</td>
								<td></td>
								<td>CCTATTACTTCAAGAAGCAGAC</td>
								<td>3′ end RACE PCR</td>
								<td>1</td>
							</tr>
							<tr>
								<td>EBLV-2 L gen end For 3</td>
								<td>M</td>
								<td></td>
								<td>11532–11552</td>
								<td>GAAGCAAGTCATACGAGGAAG</td>
								<td>3′ end RACE PCR</td>
								<td>2</td>
							</tr>
						</tbody>
					</table>
				</div>
			</div>
		</div>
		<p id="p0175">
			Sequencing Mix volume added per reaction:
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0180">
						DNA, 5.0&nbsp;μL
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0185">
						Molecular grade water, 6.0 μL
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0190">
						Big Dye Sequencing mix, 4.0 μL
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0195">
						5×sequencing buffer, 4.0 μL
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0200">
						Primer (3.2 pmol/μL), 1.0 μL
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0205">
						Total volume, 20.0 μL
					</p>
				</dd>
			</dl>
		</p>
		<p id="p0210">
			Examples of primers are shown in Table 16.1. See Table 16.2<span><span> for details of primers used for long-distance PCR to provide overlapping amplicons for European bat lyssaviruses (EBLV) </span>genome sequencing.</span>
		</p>
		<p id="p0215">
			Cover the plate with foil, sealing the lid and centrifuge briefly, to collect liquid at the bottom of the wells if necessary.
		</p>
		<p id="p0220">
			Transfer the plate to a PCR machine and run on the suitable ABI sequencing program. These can be modified to suit primers if necessary (see below).
		</p>
		<p id="p0225">
			Cycling conditions for Big Dye Sequencing reactions:
			<dl class="list">
				<dd class="list-description">
					<p id="p9055">
						<figure class="inline-figure">
							<img src="https://ars.els-cdn.com/content/image/3-s2.0-B9780128000144000160-u16-01-9780128000144.jpg" height="78" alt="">
						</figure>
					</p>
				</dd>
			</dl>
		</p>
		</div>
	</section>
	<section id="s0035" class="section-expandable">
		<h3 id="st0050" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.4. Ethanol Precipitation of Sequencing Reactions
		</h3>
		<div class="section-content">
		<p id="p0250">
			The purification of the sequencing reactions facilitates the removal of unincorporated labeled ddNTPs, which would interfere with base calling. Once the cycling is complete, remove the plate from the PCR machine and clean up the sequencing products. The Ethanol/EDTA/NaOAc clean-up method is recommended. While ethanol/EDTA can be used, the smallest PCR fragments may not be precipitated.
		</p>
		<p id="p0255">
			<span>Note: Isopropanol precipitation is NOT recommended. </span>
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0260">
						Add 2&nbsp;μL 125&nbsp;mM EDTA to each well.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0265">
						Add 2 μL 3M Sodium Acetate to each well.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0270">
						Add 50 μL of 100% Ethanol to each well.<span id="p164"></span><span id="p165"></span><span id="p166"></span>
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0275">
						Seal plate, vortex briefly and leave for 15 mins at room temperature.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0280">
						Centrifuge the plate at 3,000&nbsp;rpm for 45&nbsp;mins.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0285">
						Invert plate over sink and shake 3 times to remove supernatant.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0290">
						Rinse each well with 70&nbsp;μL 70% ethanol.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0295">
						Centrifuge the plate at 3,000&nbsp;rpm for 15 mins.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0300">
						Invert plate over sink and shake 3 times to remove supernatant.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0305">
						Keeping the plate inverted, place it on a piece of paper towel and centrifuge upside down for 15–30 seconds at 300&nbsp;rpm.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0310">
						Air dry for 15–10&nbsp;mins.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0315">
						Seal and label the 96-well plate.
					</p>
				</dd>
			</dl>
		</p>
		</div>
	</section>
	<section id="s0040" class="section-expandable">
		<h3 id="st0055" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.5. Preparing 96-Well Sample Plate(s) for the Genetic Analyzers
		</h3>
		<div  class="section-content">
		<p id="p0320">
			Spin samples down if necessary, remove sealing film.
		</p>
		</div>
	</section>
	<section id="s0045" class="section-expandable">
		<h3 id="st0060" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.6. Loading 96-Well Sample Plate(s)
		</h3>
		<div  class="section-content">
		<p id="p0325">
			The Genetic Analyzers have a safety feature, which means that no movement will occur inside the machine when the door is open. The machines will not work, and the buttons on the front of the machines will not function, unless the door is properly closed.
		</p>
		<p id="p0330">
			For the 3130xl: Press the button on the front of the machine labeled “TRAY.” Once the tray has stopped moving, the door can be opened. Inside, the machine has two positions for holding plates. Position A is on the left, and Position B is on the right. The plate assembly will only fit in one way with Column 1 furthest away from the door due to a notch in the base plate of the plate assembly. Place the plate assembly into position. Close the door and wait to see the green light appear on the front of the machine, which indicates it is properly loaded.
		</p>
		<p id="p0335">
			For the 3730: Open the stacker drawer of the Genetic Analyzer and then open the In Stack door. Place the plate assemblies into the stacker with the plate(s) orientated so that the notched corner of the plate assembly is at the rear right corner of the stacker. Up to 16 plates can be placed in the “In Stack.” Close the “In Stack” door and then the stacker draw.
		</p>
		<p id="p0340">
			Note: The In Stack is at the front of the stacker drawer with the “Out Stack” at the rear. Plates to be run are taken from the bottom of the stack and will then be at the top of the “Out Stack” once they have been run.
		</p>
		</div>
	</section>
	<section id="s0050" class="section-expandable">
		<h3 id="st0065" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.7. Checking Run Reagents
		</h3>
		<div  class="section-content">
		<p id="p0345">
			Check the polymer and buffers are within expiry dates.<span id="p167"></span>
		</p>
		<p id="p0350">
			Check there is sufficient volume of polymer in the bottle for the run. Check for bubbles in the tubes (and small tank above) leading from the POP-7. If bubbles are present, an error could occur and your run will stop. To check for bubbles, switch on the light and look inside. Remember to turn off light once checked.
		</p>
		</div>
	</section>
	<section id="s0055" class="section-expandable">
		<h3 id="st0070" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.8. Starting the Run on the 3130xl
		</h3>
		<div  class="section-content">
		<p id="p0355">
			In the Data Collection program on the PC, select from the left-hand list 3130xl/Run Scheduler in Plate View. Files which show “Processed” next to them are completed runs. Files which show “Pending” next to them are ready to be run. When you locate your file, it should be “pending.”
		</p>
		<p id="p0360">
			You will see on the right of the screen two rectangles representing the two plate positions inside the machine. The color of the rectangles means:
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0365">
						Grey – no plate(s) in holder.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0370">
						Yellow – unlinked plate(s) in holder.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0375">
						Green – linked plate(s) in holder.
					</p>
				</dd>
			</dl>
		</p>
		<p id="p0380">
			Click once on the position that relates to your plate, Position A (left) or Position B (right). It should turn green. The Plate Record will now have A or B to the left of the name, indicating linkage to the plate position.
			<dl class="list">
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0385">
						To start your run, click on the green arrow at the top left hand corner of the window.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0390">
						A dialog box will appear telling you that you are about to start processing your plate. Click OK.
					</p>
				</dd>
				<dt class="list-label">●</dt>
				<dd class="list-description">
					<p id="p0395">
						Once the run is finished, check that the run has proceeded correctly and the analyzed sequence data is available.
					</p>
				</dd>
			</dl>
		</p>
		</div>
	</section>
	<section id="s0060" class="section-expandable">
		<h3 id="st0075" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.9. Starting the Run on the 3730
		</h3>
		<div class="section-content">
		<p id="p0400">
			In the Data collection program on the PC, select from the left-hand list 3730/Run Scheduler in Plate View. Find the Plate Record file for the plate to be run first and highlight the file.
		</p>
		<p id="p0405">
			Click on Add – This will add it to the list in the Input Stack dialog box with Status as Pending. Keep adding plate records in the same order as the order they will be run, until all those to be run are in the In Stack dialog box.
		</p>
		<p id="p0410">
			Plates can be added or removed during instrument operation.
		</p>
		<p id="p0415">
			Click on Done to close the Add Plates to In Stack dialog box. Note: The Plate Record at the bottom of the list in the In Stack dialog box is marked “1” and will be run first, the next one second, etc.<span id="p168"></span>
		</p>
		<p id="p0420">
			If the sequencer is running, nothing further needs to be done. If the sequencer is not running, follow these directions: To start your run, click on the green arrow at the top left hand corner of the window. A dialog box will appear, telling you that you are about to start processing your plate. Click OK.
		</p>
		<p id="p0425">
			Once the run is finished, check that the run has proceeded correctly and the analysed sequence data are available.
		</p>
		</div>
	</section>
	<section id="s0065" class="section-expandable">
		<h3 id="st0080" class="u-h4 u-margin-m-top u-margin-xs-bottom">
			16.2.10. Analyzing Output Data
		</h3>
		<div  class="section-content">
		<div>
			<p id="p0430">
				The raw data will be generated in different formats depending on the sequencer used, but in general you would expect to obtain a chromatogram or trace from which you can derive the sequence (.seq or .fas file). From an ABI sequencer you would expect “.ab1” or “.abi” file formats (Figure 16.1), while from a Beckman sequencer you may expect a “.scf” file format.
			</p>
			<figure class="figure text-xs" id="f0010">
				<span><img src="https://ars.els-cdn.com/content/image/3-s2.0-B9780128000144000160-f16-01-9780128000144.jpg" height="271" alt="" aria-describedby="cap0010">
				<ol class="links-for-figure">
					<!--<li>
						<a class="anchor download-link u-font-sans" href="https://ars.els-cdn.com/content/image/3-s2.0-B9780128000144000160-f16-01-9780128000144.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text">Download full-size image</span></a>
					</li>		-->
				</ol>
				</span>
				<div class="captions">
					<span id="cap0010">
					<p id="sp0010">
						<span class="label">Figure 16.1</span>. <span><span>Sanger sequencing<span> of overlapping long distance PCR amplicons (forward and reverse) facilitate </span></span>genome sequencing via “Walking the Genome.” The four ABI traces of four overlapping PCR products (RV2479, 1993 EBLV-2 </span><em>M. daubentonii</em> from Switzerland) yield contiguous sequences that are superimposed (aligned) to generate a consensus sequence (top frame) determined using the Seqman Programme (DNASTAR Lasergene 10).
					</p>
					</span>
				</div>
			</figure>
		</div>
		<p id="p0435">
			A number of software packages exist for analyzing the data and generating consensus sequences from the replicate forward and reverse outputs (e.g. Seqman in Lasergene). If the sequences are good quality, they will form a contiguous sequence (called a contig). The forward and reverse contiguous sequences are aligned to generate consensus sequences (Figure 16.1).<span id="p169"></span>
		</p>
		<p id="p0440">
			If the consensus sequence saved is more than the required length, it can be trimmed to the correct length by opening in a sequence editing program (e.g. Seqman or Editseq in Lasergene 10 or MEGA<sup>13</sup>).
		</p>
		</div>
	</section>

							<div class="read-more"><a href="/science/article/pii/B9780128000144000160#s0015" class="readmore-protocol button-alternative button-alternative-primary"><span class="button-alternative-text">Read more</span></a></div>
						</div>
						<div class="relevant-methods protocols-section" id="relevant-methods">
							<h2>Relevant method sections</h2>
							<ul class="ref-list  u-font-serif">

									  <li>
										<span class="ref-title"><a href="/science/article/pii/S1050464810001580" class="relmeth-title">An inducible nitric oxide synthase (NOS) is expressed in hemocytes of the spiny lobster Panulirus argus: Cloning, characterization and expression analysis</a></span>
										<span class="ref-authors">Tania Rodríguez-Ramosac, Yamila Carpio, Jorge Bolívar, Georgina Espinosaad, Jorge Hernández-López, Teresa Gollas-Galván, Laida Ramosa, Carlos Pendón, Mario Pablo Estrad</span>
										<span class="ref-source-title">Fish &amp; Shellfish Immunology</span>
										<span class="ref-source-volume">Volume 29, Issue 3</span>
										<span class="ref-source-date">September 2010</span>
										<span class="ref-source-pages">Pages 469-479</span>
										<div class="ref-section">
											<button class="button-link">
												<span class="button-link-text">Method detail</span>
												<svg focusable="false" viewBox="0 0 92 128" width="12" height="16" class="icon icon-navigate-down preview-icon active"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg>
											</button>
										</div>
									  </li>
									  <div class="preview-body-container">
											<div class="preview-body-content">
												<div class="ArticlePreview preview-body-content">
													<div class="text-s branded preview-body">
														<h3>Material and methods</h3>
														<p><p>Lobsters between 0.5 and 0.8 kg of weight were collected by diving in the Golfo de Bataban√≥, which lies in the southwestern part of the Cuban archipelago. They were kept in 100-l tanks (two lobsters per tank) connected to a flow through seawater supply system, using biological‚ Äìmechanical filtration, seawater UV-sterilization, central aeration system, and controlled photoperiod of 12: 12 h light: obscurity. All experiments were performed after one week in these conditions.</p><p>To obtain hemocytes, the same amount of hemolymph from five lobsters was extracted using a modified Citrate-EDTA buffer (400 mM NaCl, 0.1 M glucose, 30 mM trisodium citrate, 26 mM citric acid and 20 mM EDTA, pH 4.6) as anticoagulant &#91;39]. The hemolymph was extracted from the fourth walking leg coxa using a pyrogen-free disposable syringe containing the same volume of pre-cooled anticoagulant solution. The mixture was centrifuged at 700 g for 10 min at 4 °C. The cell pellet was washed twice with anticoagulant and suspended again in anticoagulant solution.</p><div style='display:none'><p>Samples of hemocytes, digestive gland, abdominal muscle, heart, stomach, gut and gills were collected from five lobsters, frozen immediately in liquid nitrogen and stored at ‚àí80 °C prior to RNA extraction and gene expression studies. </p><<p>Hemocytes and tissues were homogenized using a tissue homogenizer (Politr√≥n UltraTurrax, Germany). Total RNA was isolated by the acid guanidinium thiocyanate-phenol-chloroform extraction method &#91;40]. </p><p>Total RNA concentration and quality were determined by UV absorbance at 260 nm, the A260/A280 ratio and by denaturing gel electrophoresis. Contaminating genomic DNA was eliminated by DNase I digestion (USB, USA).</p><p> Five micrograms of RNA from hemocytes were reverse-transcribed into cDNA using the SuperScript III RT reverse transcriptase kit (Invitrogen, USA) according to the manufacturer's instructions. Reverse transcription products were RNase H treated (Invitrogen, USA) and cDNA was purified by filtration using a Centri-sep column (Princeton Separation, Adelphia, NJ). </p><p>The primers used to obtain the cDNA sequence of P. argus NOS are described in Table 1 and Fig. 1. </p><p>Two pairs of degenerate primers (degFNOS1-degRNOS2 and degFNOS3-degRNOS4) designed from highly conserved regions in a wide variety of NOS genes in the GenBank database (http://www.ncbi.nlm.nih.gov) were used to obtain the first two fragments of NOS cDNA by PCR (Fig. 1). The analysis for primer design included the protein sequences from eight insect species, one crustacean and one mollusc: (i) insects, (a) Lepidopteran: Manduca sexta (GenBank AAC61262), Bombyx mori (REFSEQ NP_001036963); (b) dipteran: Drosophila melanogaster (GenBank AAF25682), A. stephensi (GenBank AAC68577); (c) hymenopteran: Apis mellifera (REFSEQ NP_001012980); (d) hemipteran:Rhodnius prolixus (GenBank AAB03810), (e) orthoptera: Gryllus bimaculatus (DBJ BAH14964); (f) coleoptera: Luciola cruciata (DBJ BAF63161), (ii) crustaceans, decapoda: G. lateralis (GenBank AAT46681); (iii) mollusc, Aplysia californica, AF288780. The ClustalW software (http://www.ebi.ac.uk/clustalw/index.html) was used for the alignment of the deduced amino acid sequences. These sequences were also used to draw an interspecific phylogeny of NOS proteins, in which two other crustacean NOS, recently cloned from the cladoceran D. magna (GenBank ACQ55298 and ACQ55299) were also included. </p><p>PCR amplifications using the above degenerated primers were performed under the following conditions: initial denaturation at 95 °C for 5 min, 40 amplification cycles (denaturing at 95 °C for 30 s, annealing temperature at 42 °C, and extension at 72 °C for 1 min), and a final extension step at 72 °C for 5 min. Master Mix PCR (Promega, USA) was used for amplifications. The PCR products were separated using 0.8% agarose gel electrophoresis, and purified from gel slices using a QIAquick Gel Extraction mini kit (Qiagen, Germany). The purified PCR products were cloned into pCR 4-TOPO vector (Invitrogen, USA) following the manufacturer's instructions and the recombinant plasmids was isolated using the Minipreps DNA purification system (Promega, USA). The nucleotide sequences from five clones of each fragment were then determined by the standard dye terminator chemistry using the Big Dye Terminator v3.1 Cycle Sequencing kit (Applied Biosystems, USA) in an ABI PRISM 3100 Genetic Analyzer (Applied Biosystems, USA). </p><p>The consensus sequences derived from each fragment were used to design two specific primers (espFNOS1- espRNOS2) to obtain an overlapping PCR fragment (Fig. 1), using the Pfu DNA polymerase (Promega, USA). </p><p>For amplification, the following conditions were used: initial denaturing at 95 °C for 5 min, 35 amplification cycles (denaturing at 95 °C for 30 s, annealing temperature at 55 °C, and extension at 74 °C for 3 min), and a final extension step at 74 °C for 5 min. PCR products were purified, cloned and sequenced as described above. </p><p>The characterization of 5‚Ä≤ and 3‚Ä≤ ends of NOS was carried out by RACE. </p><p>To obtain the 3‚Ä≤ end the first-strand cDNA synthesis reaction contained the components of the Promega Reverse Transcription System kit (Promega, USA) with the exception of a modified oligo dT (OdTmod). For the first round of PCR amplification, a pair of gene-specific primers (3FNOS1-3RNOS2) was used. Master Mix PCR (Promega, USA) was used. PCR consisted of an initial denaturing step at 95 °C for 5 min, 35 amplification cycles (denaturing at 95 °C for 30 s, annealing temperature at 63 °C, and extension at 72 °C for 1 min), and a final extension step at 72 °C for 5 min. For a nested PCR, 0.1 Œºl of the first round PCR product and a pair of gene-specific nested primers (3FNOS3-3RNOS2) were used. PCR conditions were similar to those in the first reaction. DNA fragments were purified, cloned and sequenced as described above. To obtain the 5‚Ä≤ end of the NOS gene, a homopolimeric dC tail was added to the 5‚Ä≤ end of cDNA using Terminal Deoxynucleotidyl Tranferase (Invitrogen, USA), and the tailed cDNA directly used for PCR amplification assays, using the KOD HOT Start DNA Polymerase (Novagen, USA). For the first round of PCR, a gene-specific reverse primer (5RNOS2) and an Oligo dG primer (OdGNot1) were used. PCR consisted of an initial denaturing step at 95 °C for 2 min, 35 amplification cycles (denaturing at 95 °C for 30 s, annealing temperature at 60 °C, and extension at 68 °C for 2.5 min), and a final extension step at 68 °C for 10 min. The second and third round of PCR amplifications (semi-nested) were performed with the Oligo dG primer and specific-gene reverse primers (5RNOS3, 5RNOS4). Recombinant clones were cloned and sequenced as described above.</p><p> Standard DNA and protein sequence bioinformatics analyses were performed using BLASTn and BLASTx searches (http://blast.ncbi.nlm.nih.gov). Multiple sequence alignments of deduced amino acids sequence were performed using the CLUSTALw2 (1.4) algorithm (http://www.ebi.ac.uk/Tools/clustalw2/). Conserved domains were investigated using the Conserved Domain Search tool at NCBI (http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi). </p><p>One and half micrograms of total RNA from gills, heart, stomach, digestive gland, abdominal muscle, gut and hemocytes were reverse-transcribed into cDNA using the iScript cDNA synthesis kit (Bio-Rad, USA) to a final volume of 30 Œºl, according to the manufacturer's instructions. </p><p>The primers used for the analysis of the expression of NOS gene from tissues and hemocytes of P. argus are described in Table 1. </p><p>Expression analysis was performed by Real time qPCR. Specific-gene primers (qFNOS1 - qRNOS2) were designed according to the P. argus NOS sequence obtained in this study (GenBank ACZ60615) to amplify an 85 bp fragment from non-LPS-treated lobster tissues and hemocytes. Five lobsters were used and all reactions were performed in triplicate.</p><p> Amplification of NOS cDNA samples was carried out in a MiniOpticom Real Time PCR System (Bio-Rad, USA), using the 2X Sensimix dU SYBR Green Kit (Quantace, UK) in 20 ŒºL final volume and the following conditions: 10 min at 94 °C for enzyme activation and initial denaturing, 35 cycles of 15 s at 95 °C, 30 s at 64 °C and 10 s at 72 °C. Five animals per tissue, three replicates per animal and cDNA from 200 ng of total RNA per replicate were assayed. </p><p>NOS expression was quantified using absolute quantification method. The amplification of 90 bp fragment of Beta actin (Œ≤-act) gene and an 108 bp fragment of the elongation factor alpha (EFŒ±) gene were amplified in order to evaluate the presence of good quality cDNA in samples. The primers for the amplification of a EFŒ± cDNA fragment (qFEFŒ±1- qREFŒ±2) were designed according to the P. argus EFŒ± cDNA sequence previously reported (GenBank GU33803) whereas for the amplification of a Œ≤-act cDNA fragment, specific gene-sequence primers (qFŒ≤act1- qRŒ≤act2) were designed guided by a partial P. argus Œ≤-act cDNA that was cloned and sequenced in this work (GenBank ACZ60616). </p><p>To generate the standard curve(s) for quantitative determinations and to access amplification efficiency, five 10-fold serial dilutions of the linear pCR 4-TOPO vector plasmid containing the target/reference sequence were prepared. </p><p>After each reaction, a melting curve was performed to confirm that a single PCR product was amplified. Quantitative analyses were performed using MJ Opticon Monitor‚Ñ¢ Analysis Software v.3.1 (Bio-Rad, USA). </p><p>All quantification experiments included an NTC (no template control), containing all the components of the reaction except for the template, and all Reverse Transcriptase-PCR experiments included a negative control to test for contaminating DNA. </p><p>To evaluate the effect of Escherichia coli LPS on in vitro activity and expression of P. argus NOS in hemocytes, three experiment were performed using replicated block designs. </p><p>In the first experiment three LPS concentrations were used: 0.00, 0.25 and 0.50 mg/ml. The three time treatments were 1, 2 and 3 h after the beginning of the experiment. For this assay, 15 ml of hemolymph per lobster were extracted from five different animals. The hemocytes were obtained as described above and kept in the anticoagulant solution. Each lobster was sampled at the beginning of the experiment (t = 0). In the control, the LPS was substituted by 400 mM NaCl. </p><p>The experiment was performed using 96-well plates. Each well had around 1million total hemocytes and 200 Œºl of final volume. After LPS exposure, supernatants were used to measure nitrites as a mean of NOS activity in vitro. To achieve this objective nitrates were first converted to nitrites using the nitrate reductase enzyme (Boheringer, Spain). After that, the colorimetric Griess reaction for measuring inorganic nitrites was performed. This method is based on the reaction of nitrites with sulfanilamide and N-1-(naphtyl) ethylenediamine, which generates a purple-azo dye product that can be monitored by a spectrophotometer at 540 nm &#91;41]. </p><p>The best dose of E. coli LPS (0.50 mg/ml) and time of exposure (1 h) from the first experiment were selected in a second in vitro experiment that was performed to evaluate NOS activity with and without the NOS inhibitor: Nœâ-monomethyl-l-arginine (l-NMMA). The 1.5 mM and 3.0 mM doses of L-NMMA were assayed. In the first two experiments the reactions were performed in duplicate and five lobsters were used individually. </p><p>A third in vitro experiment was performed to evaluate NOS expression by real time qPCR. The samples were collected at 20 and 40 min after the exposure to LPS, then these and the t = 0 sample were assayed. The dose of 0.50 mg/ml of E. coli LPS was used. Hemocyte collection and RNA purification were performed as described above. The qPCR amplification conditions were the same as described for NOS tissue expression. </p><p>NOS expression in hemocytes was quantified using relative quantification &#91;42]. EFŒ± and Œ≤-act genes were used as references. Total RNA samples taken from five animals per treatment as well as three replicates per animal were used. For NOS amplification cDNA from 200 ng of total RNA per replicate were assayed, whereas cDNA from 50 ng of RNA were enough for EFŒ± and Œ≤-act amplifications. The qPCR conditions for the reference genes were the same as above described for NOS, except that an annealing step at 60 °C was used for EFŒ±. The primers used for relative qPCR were the same as described above for the analysis of NOS expression pattern and are shown in Table 1. </p><p>Since the amplification efficiencies were nearly equivalent, the comparative method (ŒîŒîCt), which relies on comparing differences in Ct values, was used. The values of Ct were first normalized using the Ct value of the reference genes, and then they were normalized using the expression at time 0. </p><p>STATISTICA 6.0 software (StatSoft, Inc. 2001, Tulsa, OK, USA) was used for statistical analyses. Two-way ANOVA, followed by Newman-Keuls test was used to test for significant differences. Data are reported as the mean ¬± standard deviation. The level of significance was set at ‚â§0.05 for all analyses.</div></p>
													</div>
												</div>
												<div class="read-more"><a href="/science/article/pii/S1050464810001580" class="readmore-relmethod button-alternative button-alternative-primary"><span class="button-alternative-text">Read more</span></a></div>
											</div>
										</div>


									  <li>
										<span class="ref-title"><a href="/science/article/pii/S1050464810001579" class="relmeth-title">Attempts at producing a hybridised Penaeus mondon cell line by cellular fusion</a></span>
										<span class="ref-authors">Kerry Claydon; Katrina G.Roper; Leigh Owens</span>
										<span class="ref-source-title">Fish &amp; Shellfish Immunology</span>
										<span class="ref-source-volume">Volume 29, Issue 3</span>
										<span class="ref-source-date">September 2010</span>
										<span class="ref-source-pages">Pages 539-543</span>
										<div class="ref-section">
											<button class="button-link">
												<span class="button-link-text">Method detail</span>
												<svg focusable="false" viewBox="0 0 92 128" width="12" height="16" class="icon icon-navigate-down preview-icon active"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg>
											</button>
										</div>
									  </li>
									  <div class="preview-body-container">
											<div class="preview-body-content">
												<div class="ArticlePreview preview-body-content">
													<div class="text-s branded preview-body">
														<h3>Material and methods</h3>
														<p><p>P. monodon were collected from wild stocks off the coast of northern Queensland, Australia, or from a commercial farm located at Innisfail, Queensland, Australia. Following collection, P. monodon were maintained at 28¬±3°C at James Cook University in 1t Reln¬© recirculating tanks with biological filtration. Two immortal cell partners, Epithelioma papulosum cyprinid (EPC) and Spodoptera frugiperda pupal ovarian cells (Sf9) were investigated in cell fusion experiments. Cells were maintained in 25cm2 flasks containing 10ml of Leibovitz 15 (L-15) (Gibco, Brisbane, Australia) or tissue culture medium 100 (TC-100) (Gibco, Brisbane, Australia) for the maintenance of the EPC and Sf9 cells respectively, and supplemented with 10% foetal bovine serum (FBS) (Trace Biosciences, Castle Hill, Australia). The cells were observed daily with an Olympus IMT-2 inverted phase-contrast microscope, and once a monolayer formed, cells were trypsinised, split and re-seeded.</p><div style='display:none'><p> A total of 300Œºl of haemolymph from each P. monodon was extracted using a 25-gauge needle inserted into the dorsal thoracic chamber. A 50% dilution with Alsever's anticoagulant (27mM Na3C6H5O7, 336mM NaCl, 226mM glucose, 9mM EDTA) was used to avoid clotting. After brief centrifugation (500√óg for 5min), the haemocytes were resuspended in L-15 medium with 10% FBS supplemented with antibiotics (penicillin 200IU/ml, streptomycin 200Œºg/ml, kanamycin 80Œºg/ml, polymyxin B 5Œºg/ml, and fungizone¬Æ 2.5Œºg/ml). Haemocytes from one animal were used for each fusion. EPC and Sf9 cells at 4√ó106 cells per flask were suspended in 20ml culture medium without FBS. The cells were treated with actinomycin D (0.25Œºg/ml) and puromycin (2√ó10‚àí4M) &#91;17]. Culture flasks were incubated at 28°C for 3h and rocked gently every 30min to mix the cell suspension. To determine the amount of viable cells, equal volumes of a cell suspension and 0.2% trypan blue solution were mixed, loaded into the counting chamber of a haemocytometer, and examined using an Olympus IMT-2 inverted phase-contrast microscope. The viable cells, which were not stained blue, were counted and converted into viable cells/ml. The three flasks of biochemically-blocked cells were combined to provide a total of 1.20√ó106 viable cells. The cells were washed twice with culture medium without FBS and pelleted in a 10ml tube. Freshly harvested prawn haemocytes were added to the pelleted biochemically-blocked cells in a 1:1 ratio. The tube was made up to 10ml with culture medium and then centrifuged. The supernatant was carefully removed and pre-warmed PEG-4000 (Koch-Light) (50%) was added dropwise over a 45s with constant swirling in a 27°C waterbath. The cells were left for 60s and then 10ml of culture medium was added dropwise to dilute out the PEG solution. The cells were gently inverted to mix the PEG and left undisturbed for a further 5min. The cells were pelleted, then resuspended in 10ml of culture medium with 20% FBS. To obtain single cell colonies, the cells suspension was seeded out over a 96-well plate at 200Œºl per well. Columns 2 through to 7 were used, with columns 1 and 8 containing culture medium only to help protect the cell-containing wells from drying out. The plate was sealed inside a plastic container and incubated at 28°C. Cells were examined on days 1 and 6 post-fusion and 100Œºl of medium exchanged. On day 14 post-fusion, cells of viable appearance were transferred to 24-well plates. Once confluent in the 24-well plates (generally from 7 d post-transfer), the cells were transferred to 25cm2 flasks and supported in culture medium supplemented with 10% FBS. When confluent in the flasks, cells were trypsinised and stored in liquid nitrogen, and later tested for crustacean components. To verify that crustacean DNA was incorporated into the fused cells, two PCRs were conducted using arthropod specific 18S rRNA primers 143F/143R &#91;18] and primers designed on the P. monodon haemocyanin gene (GenBank accession no. AF431737) &#91;19]. Once a confluent monolayer of cells had formed, cells were trypsinised and pelleted using slow centrifugation (500√óg for 5min). DNA was extracted from the cells using a High Pure PCR Template Preparation Kit (Roche Sciences, Brisbane, Australia). For PCR analysis, 1Œºl of the DNA template (50-150ng) was added to 49Œºl of PCR reaction mixture consisting of MBI fermentas PCR buffer (750mM Tris-HCl pH 8.8, 200mM (NH4)2SO4, 0.1% Tween20) 200ŒºM dNTPs, 1ŒºM of each primer, 1.5mM MgCl2 and 1U Taq polymerase (MBI Fermentas, Hanover, USA). The arthropod 18S rRNA primer sequences and methodology are described elsewhere &#91;18]. For the haemocyanin PCR, primer sequences were H-299, 5‚Ä≤ -TCG-ACA-GGG-CTG-GTA-ATC-ATA-3‚Ä≤ and H-594, 5‚Ä≤-CCG-GCA-AAT-GTT-AGT-TCT-3‚Ä≤ with an expected amplicon of 295bp. The optimised PCR profile consisted of an initial 3min denaturing at 95°C followed by 40 cycles of 95°C for 1min, 59°C for 1min, and 72°C for 1min and a final 5min extension at 72°C. To analyse the PCR products, 10Œºl of the PCR reaction mixture was used in electrophoresis on 1.5% agarose gels containing 1mg/ml ethidium bromide and visualised under ultraviolet transillumination. Amplicons selected for DNA sequencing were purified from the agarose gel using QIAquick Gel Extraction Kit (Qiagen, Melbourne, Australia) and sent to Macrogen Corporation (Seoul, South Korea). Analysed sequences were then compared to available databases using the Basic Local Alignment Search Tool (BLAST) to determine approximate phylogenetic affiliations. A monoclonal antibody (MAb) developed by Ainsworth &#91;20], was used to assess the presence of crustacean proteins in hybrid cell populations. After assessing contamination and responsiveness of the antibodies, antibody 1.9B2-5 was selected for immunostaining of the hybrid cell lines which was categorised as a type I antibody with general haemocytic specificity, staining approximately 78% of P. monodon haemocytes &#91;20]. Polyclonal antibodies (PAb) were also used to assess crustacean proteins in hybrid cell populations. In contrast to monoclonal antibodies, which are derived from a single cell line, a PAb contains a mixture of immunoglobulin molecules secreted against many antigens. The polyclonal antibodies used in the present study were initially developed in chickens against gill-associated virus (GAV) of P. monodon &#91;21]. This particular PAb was found to lack reactivity against GAV and was instead concluded to have a presumptive prawn cell origin which therefore may contain an array of non-specific proteins that react with P. monodon cells, making it an ideal candidate to use against the developed hybrid cells. For immunohistochemisty (IHC) cells were spun onto positive charged microscope slides and fixed in acetone for 20min. Endogenous peroxidase activity was blocked by adding 0.3% hydrogen peroxide in methanol to each slide for 30min. After washing with phosphate buffered saline with 1% Tween20 (PBS-T), slides were incubated with either the MAb or PAb (1:100) for 1h at 37°C in a humid enclosed chamber. Slides were then subjected to three 5min washes with PBS-T, and incubated with the secondary antibody (anti-mouse horseradish peroxidase (HRP) conjugate, 1:120, Jackson Immuno Research, West Grove, USA) in a humid chamber for 1h at RT. After washing the slides in three 5min washes with PBS-T, slides were incubated with fresh diaminobencidine substrate for 10min, washed briefly in water, and counter-stained with haematoxylin for 2min. Following a brief water wash, slides were then exposed to Scott's tap water substitute for 30 s before final water wash. Slides were then dehydrated through grades of ethanol, cleared with xylene, mounted and examined using an Olympus BH-2 microscope. For quantitative analysis of crustacean proteins, an indirect ELISA was also developed using anti-chicken polyclonal antibodies &#91;21]. Hybrid cells were trypsinised from 25cm2 flasks and centrifuged at 500√óg for 5min. The cell count was determined by haemocytometer. Approximately 1√ó106 cells were lysed by adding 125Œºl of CellLytic MT reagent (Sigma, Brisbane, Australia). After 15min of incubation with lysis reagent, cells were centrifuged at 15‚Äà000√óg for 10min, and the protein containing supernatant removed for use as an antigen in the ELISA. Cell supernatant was diluted (1:1) in bicarbonate-carbonate coating buffer (TropBio, Townsville, Australia); 50Œºl was loaded into each well of a 96-well U-bottom plate (Sarstedt, Adelaide, Australia), and incubated overnight in a humid environment. Following coating, 150Œºl of post-coating buffer (TropBio Townsville, Australia) was added and the plate incubated for 1h before washing three times in wash buffer (Trop Bio, Townsville, Australia). Fifty Œºl of diluted (1:80) PAb were added to each well, incubated for 1h in a humid environment and washed three times with wash buffer. The secondary conjugated antibody was diluted (1:120), and 50Œºl were added to each well. Following 1h incubation, the plate was washed six times with wash buffer. Finally, 100Œºl of ABTS peroxidase substrate solution were added to each well, and the plate was again incubated for 1h. Colour formation was determined with dual absorbance readings at 415nm and 490nm, using a Labsystems Multiskan EX ELISA plate reader (Pathtech, Preston, Australia). All incubation steps were performed at RT. A second ELISA was performed to detect the crustacean protein tropomyocin by sending samples to a commercial company, Elisa Systems (Windsor, Brisbane, Australia). Hybrid cells were also exposed to two prawn viruses and quantification levels monitored. The viruses were the Australian Penaeus merguiensis densovirus (PmergDNV) and the Australian strain of infectious hypodermal and haematopoietic necrosis virus (IHHNV), both of which are known to infect P. monodon. The more lethal viruses such as white spot syndrome virus (WSSV) and/or yellow head virus (YHV) might have been more suitable for viral inoculation of cell cultures, but both viruses are exotic to Australia and therefore unavailable. Similar methods were used to isolate and purify PmergDNV and IHHNV from prawn tissue. For PmergDNV, 25 hepatopancrei were removed from PCR-positive P. merguiensis &#91;22]. For IHHNV, 100 pleopods were removed from 10 PCR-positive P. monodon &#91;23]. Hepatopancrei or pleopods were homogenised in 250ml TN buffer (0.02M Tris-HCl, 0.4M NaCl, pH 7.4) and centrifuged at 4500 x g for 15min to pellet tissue debris. The supernatant was re-centrifuged at 13‚Äà000√óg for 15min. The liquid supernatant was collected and vacuum filtered through 0.45Œºm filter membrane. Ultracentrifugation was then performed according to the method described in &#91;24]. The final viral pellet was resuspended in 3ml of TN buffer and stored at ‚àí80°C until required. To verify the presence of viral DNA, a DNA extraction was performed on 200Œºl of purified virus using the previously described primers and PCR methodology for PmergDNV &#91;22] and IHHNV &#91;23]. Duplicate 75cm2 flasks of hybrid cells, plus control cell lines of EPC and Sf9 were grown to 60-80% confluency. Following removal of the culture media, each flask was inoculated with either 100Œºl of purified virus or 100Œºl of TN buffer. The flasks were gently agitated for 20min to promote viral absorption &#91;25]. Then, 15ml of fresh culture medium (pH7.4) with 5% FBS was added, without removing inoculum. The cells were incubated at 28°C and monitored once a day for 14 d for cytopathic effect (CPE). One ml of cell culture medium was collected at days 0, 4, 7, 10 and 14 d post-exposure and replaced with equal volume of fresh medium. Two hundred Œºl of these samples were used to perform a DNA extraction. Extracted DNA was used in quantitative real time PCR to determine if viral replication was evident. Real time PCR was performed in accordance to the developed protocols for PmergDNV &#91;22] and Australian IHHNV &#91;26]. To accommodate small quantification differences, if the number of viral copies present at the end of the experiment (day 14) was approximately one log different to the number on day 0 (i.e. &gt;10 fold difference), viral copies were considered to increase/decrease. A stable value was reported otherwise. Serial dilutions from previously constructed PmergDNV and IHHNV plasmids were used to determine a standard linear relationship for quantification.</div></p>
													</div>
												</div>
												<div class="read-more"><a href="/science/article/pii/S1050464810001579" class="readmore-relmethod button-alternative button-alternative-primary"><span class="button-alternative-text">Read more</span></a></div>
											</div>
										</div>


									  <li>
										<span class="ref-title"><a href="/science/article/pii/S1050464810001555" class="relmeth-title">Identification of upregulated immune-related genes in Vibrio harveyi challenged Penaeus monodon postlarvae</a></span>
										<span class="ref-authors">S. Nayak, S.K. Singh, N. Ramaiah, R.A. Sreepada</span>
										<span class="ref-source-title">Fish &amp; Shellfish Immunology</span>
										<span class="ref-source-volume">Volume 29, Issue 3</span>
										<span class="ref-source-date">September 2010</span>
										<span class="ref-source-pages">Pages 544-549</span>
										<div class="ref-section">
											<button class="button-link">
												<span class="button-link-text">Method detail</span>
												<svg focusable="false" viewBox="0 0 92 128" width="12" height="16" class="icon icon-navigate-down preview-icon active"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg>
											</button>
										</div>
									  </li>
									  <div class="preview-body-container">
											<div class="preview-body-content">
												<div class="ArticlePreview preview-body-content">
													<div class="text-s branded preview-body">
														<h3>Material and methods</h3>
														<p><p>Ten-day old postlarvae (PL) of P. monodon (Crustacea, Decapoda) were obtained from a commercial hatchery and, held in the laboratory in a 500 L tank filled with seawater under continuous aeration. The PL were fed twice daily with a commercial artificial feed and, maintained throughout the 3 day experimental period under aeration at 25 °C and 29 psu salinity.</p><p> Replicates of two glass containers of 2-L capacity were filled with 1 L filtered autoclaved seawater. PL14 of P. monodon (60 nos/container) were acclimatized for a day under aeration. A strain of V. harveyi D3 (identity confirmed through partial sequencing of its 16 S rRNA gene) found to cause lethargy or mortality in all 3 trial runs was used for challenging the PL. Broth cultures of this strain grown for 24 h in seawater nutrient broth were used for the experimental challenge. Cells were harvested by centrifugation and re-suspended in sterilized seawater and the PL were 'bath' challenged by adding the harvested cells to attain 2.5-3.2 (X 10<sup>6</sup>) cells ml<sup>-1</sup> in the experimental containers for 48 h. Two glass containers filled with 1 L filtered autoclaved seawater holding ~60 PL with no added bacterial cells served as experimental controls.</p><div style='display:none'><p> Total RNA was isolated from both challenged and control populations. PL were first washed in sterile seawater and, 35-40 whole PL (weighing ~500 mg after blotting) were used for RNA extraction using Trizol Reagent (TRI, Sigma, USA). The concentration and purity of RNA samples were determined spectrophotometrically by measuring the absorbance at 260 nm. An aliquot of RNA sample was run on 1% formaldehyde agarose gel for checking structural integrity. Later, poly A+ RNA was extracted from the control and tester sets of PL using Oligotex mRNA Kit (Qiagen, Valencia, CA). SSH technique &#91;4] was performed to generate subtracted cDNA library between V. harveyi-challenged (tester) and control (driver) PL using the PCR-select cDNA subtraction kit (Clontech, Palo Alto, CA) according to the manufacturer's protocol. Briefly, tester and driver cDNA products were purified by ethanol precipitation, and then digested with Rsa I at 37 °C overnight to obtain shorter, blunt-ended fragments.</p><p> Following this, adaptor 1 and adaptor 2R were ligated to each strand of tester cDNA. Then, the ligated tester cDNA strands were hybridized at 68 °C for 8 h with an excess of driver cDNA after denaturation at 98 °C for 90 s. After the first hybridization, two sets of adaptor ligated cDNA were mixed together without denaturation and hybridized again with freshly heat-denatured driver cDNA for 20 h at 68 °C. The resulting mixture was diluted 1:50 and amplified by two rounds of PCR to enrich desired cDNA (i.e., containing both adaptors) through exponential amplification.</p><p> The primary PCR was performed with flanking primers against adaptor 1 and adaptor 2R and, the amplified products were used as template in the secondary PCR with nested primers. The efficiency of SSH was evaluated by another PCR protocol by comparing the abundance of Œ≤-actin gene in the subtracted and unsubtracted populations. The secondary PCR products were purified and cloned into pGEM-T easy vector (Promega, Madison, USA) and transformed into E.coli Top10 electrocompetent cells through electroporation. The transformed mixture was plated on agar plates containing ampicillin, X-gal, and IPTG. Randomly selected white colonies were single pass sequenced. All raw sequences were manually screened for vector sequences using Vecscreen available at the http://www.ncbi.nlm.nih.gov/VecScreen/VecScreen.html. After removal of vector and exclusion of poor quality regions from the sequences, the ESTs were analyzed using BLASTX and BLASTN algorithms. Database searches were limited to ESTs gt; 100 bases in length. Blast matchings with E-value &lt;10‚àí3 were considered as significant and only these were considered for categorization of ESTs into different groups. To further confirm the changes in expression induced by bacterial challenge, real time gene expression analyses of six different immune-related transcripts were carried out. Total RNA was treated with DNaseI at 37 °C for 30 min to ensure genomic DNA-free RNA preparation. The first strand cDNA was synthesized from 1 Œºg of DNA-free total RNA with oligo-dT primers and Superscript III reverse transcriptase (Invitrogen). Relative expression levels of hemocyanin, ferritin, TCTP, lysozyme, penaeidin and MnSOD were measured using the primer pairs listed in Table S1. Shrimp Œ≤-actin gene was used as an internal control during all the qPCR runs. The relative fold of induction was calculated using comparative CT method for relative quantification. The amplifications were carried out in a 96 well plate in a total volume of 25 Œºl using 4 Œºl of cDNA, 200 nM of each gene-specific primers pair with 2X SYBR Green Jump Start Taq Ready Mix (Sigma) in an ABI 7500 Real-Time PCR system. Cycling parameters were 95 °C for 15 min, followed by 40 cycles of 95 °C for 15 s and 60 °C for 1 min. The specificity of PCR products was confirmed by dissociation curve analysis performed at the end of qPCR by continuously heating from 55 °C to 95 °C with an increment of 0.1 °C. Data were analyzed using the ABI 7500 software version 2.0.1. The relative fold of induction was determined by ŒîŒî CT method &#91;6]. The fold change was calculated using the formula 2‚àíŒîŒî CT.</div></p>
													</div>
												</div>
												<div class="read-more"><a href="/science/article/pii/S1050464810001555" class="readmore-relmethod button-alternative button-alternative-primary"><span class="button-alternative-text">Read more</span></a></div>
											</div>
										</div>


									</ul>
						</div>
						<div class="related-articles protocols-section" id="related-content">
							<h2>Related content</h2>
							<ul class="ref-list">

								<li>
									<span class="ref-title"><a href="/science/article/pii/S016777991830204X" class="reviewart-title">Single-Molecule Sequencing: Towards Clinical Applications</a></span>
									<span class="ref-authors">Adam Ameur, Wigard P. Kloosterman, Matthew S. Hestand</span>
									<span class="ref-source-title">Trends in Biotechnology</span>
									<span class="ref-source-volume">Volume 37, Issue 1</span>
									<span class="ref-source-date">January 2019</span>
									<span class="ref-source-pages">Pages 72-85</span>
								</li>

								<li>
									<span class="ref-title"><a href="/science/article/pii/B9780128011058000059" class="reviewart-title">Chapter Five: Splicing in the Human Brain</a></span>
									<span class="ref-authors">Ammar Zaghlool, Adam Ameur, Lucia Cavelier, Lars Feuk</span>
									<span class="ref-source-title">International Review of Neurobiology</span>
									<span class="ref-source-volume">Volume 116</span>
									<span class="ref-source-date">2014</span>
									<span class="ref-source-pages">Pages 95-125</span>
								</li>

								<li>
									<span class="ref-title"><a href="/science/article/pii/S2152265011002837" class="reviewart-title">Transcriptome Sequencing Reveals Novel Mutations and Differential Gene Expression in Stereotyped Subsets of Chronic Lymphocytic Leukemia</a></span>
									<span class="ref-authors">Lesley-Ann Sutton, Larry Mansouri, Rebeqa Gunnarsson, Adam Ameur, Sean Hooper, Markus Rasmussen, Anders Isaksson, Ulf Gyllensten, Richard Rosenquist</span>
									<span class="ref-source-title">Clinical Lymphoma Myeloma and Leukemia</span>
									<span class="ref-source-volume">Volume 11, Supplement 2</span>
									<span class="ref-source-date">October 2011</span>
									<span class="ref-source-pages">Pages s148-s149</span>
								</li>

								<li>
									<span class="ref-title"><a href="/science/article/pii/B9780124186873000148" class="reviewart-title">Chapter Fourteen: Sanger Dideoxy Sequencing of DNA</a></span>
									<span class="ref-authors">Sarah E. Walker, Jon Lorsch</span>
									<span class="ref-source-title">Methods in Enzymology</span>
									<span class="ref-source-volume">Volume 529</span>
									<span class="ref-source-date">2013</span>
									<span class="ref-source-pages">Pages 171-184</span>
								</li>

								<li>
									<span class="ref-title"><a href="/science/article/pii/S0002929717301593" class="reviewart-title">Whole-Genome Sequencing Coupled to Imputation Discovers Genetic Signals for Anthropometric Traits</a></span>
									<span class="ref-authors">Ioanna Tachmazidou, Dániel Süveges, Josine L. Min, Graham R.S. Ritchie, Julia Steinberg,...,Eleftheria Zeggini</span>
									<span class="ref-source-title">The American Journal of Human Genetics</span>
									<span class="ref-source-volume">Volume 100, Issue 6</span>
									<span class="ref-source-date">1 June 2017</span>
									<span class="ref-source-pages">Pages 865-884</span>
								</li>

								<li>
									<span class="ref-title"><a href="/science/article/pii/S014067361360433X" class="reviewart-title">A whole-genome sequencing approach to targeting Mycobacterium tuberculosis outbreak management</a></span>
									<span class="ref-authors">Dr Timothy Walker MBBS, Camilla L PhD, Ruth H Harrell PhD, Jason T Evans PhD, Georgia Kapatai PhD, Martin J Dedicoat PhD, ..., Prof Tim E Peto DPhil</span>
									<span class="ref-source-title">The Lancet</span>
									<span class="ref-source-volume">Volume 380, Supplement 3</span>
									<span class="ref-source-date">23 November 2012</span>
									<span class="ref-source-pages">Page s77</span>
								</li>

							</ul>
						</div>
					</div>
					<div class="vertical-divider">
						<span class="divider"></span>
					</div>
					<div class="protocols-sidebar">
						<div class="protocols-index">
							<ul class="index">
								<li><a href="#relevant-methods" class="icon-link jump-link"><span class="index-link-icon"><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-arrow-down icon-link-icon"><path d="m98.4 55.96l-40.4 40.4v-84.36h-1e1v84.08l-40.12-40.12-7.06 7.08 52.32 52.32 52.34-52.32-7.08-7.08"></path></svg></span><span class="icon-link-text relevant-method-jump">Relevant method sections</span></a></li>
								<li><a href="#related-content" class="icon-link jump-link"><span class="index-link-icon"><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-arrow-down icon-link-icon"><path d="m98.4 55.96l-40.4 40.4v-84.36h-1e1v84.08l-40.12-40.12-7.06 7.08 52.32 52.32 52.34-52.32-7.08-7.08"></path></svg></span><span class="icon-link-text review-articles-jump">Related content</span></a></li>
								<li><a href="#related-researchers" class="icon-link jump-link"><span class="index-link-icon"><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-arrow-down icon-link-icon"><path d="m98.4 55.96l-40.4 40.4v-84.36h-1e1v84.08l-40.12-40.12-7.06 7.08 52.32 52.32 52.34-52.32-7.08-7.08"></path></svg></span><span class="icon-link-text review-articles-jump">Key researchers</span></a></li>
							</ul>
						</div>
						<div class="protocols-section videoContent">
								<h3>Related videos</h3>
								<div class="src-content videoWrapper">
									<iframe width="400" height="275" src="https://www.youtube.com/embed/EzhJije2xhM?" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
								</div>
								<div class="caption"><a href="" class="related-video-caption"></a></div>
						</div>
						<div class="related-researchers protocols-section" id="related-researchers">
							<h2>Key researchers</h2>

							<div class="experts">
								<div class="expert-image"><a href="https://www.scopus.com/authid/detail.uri?authorId=9333331600" class="researcher-image"><img 	src="/assets/protocols-plus/assets/page/Ameur.jpeg"/></a></div>
								<div class="expert-info-box">
									<span class="expert-name"><a href="https://www.scopus.com/authid/detail.uri?authorId=9333331600" class="researcher-name">Adam Ameur</a></span>
									<span class="expert-where"><p>Science for Life Laboratory </p><p>Uppsala, Sweden</p></span>
								</div>
								<div class="expert-rating-box">
									<h3>Author Ratings</h3>
									<span class="rating"><span class="value">22</span><label>h-index</label></span>
									<span class="rating"><span class="value">4973</span><label>Citations</label></span>
									<span class="rating"><span class="value">58</span><label>Documents</label></span>
								</div>
							</div>

							<div class="experts">
								<div class="expert-image"><a href="https://www.scopus.com/authid/detail.uri?authorId=9332661400" class="researcher-image"><img 	src="/assets/protocols-plus/assets/page/WalkerSarah.jpg"/></a></div>
								<div class="expert-info-box">
									<span class="expert-name"><a href="https://www.scopus.com/authid/detail.uri?authorId=9332661400" class="researcher-name">Sarah E. Walker</a></span>
									<span class="expert-where"><p>University at Buffalo, State University of New York </p><p>Buffalo, United States</p></span>
								</div>
								<div class="expert-rating-box">
									<h3>Author Ratings</h3>
									<span class="rating"><span class="value">11</span><label>h-index</label></span>
									<span class="rating"><span class="value">538</span><label>Citations</label></span>
									<span class="rating"><span class="value">19</span><label>Documents</label></span>
								</div>
							</div>

						</div>
					</div>
				</div>
			</div>
		</section>
	  </main>
	  <div id="back2TopBox" style="display:none;"><a id="back2Top" title="Back to top" href="#">&#x2191;</a><span>To Top</span></div>
	</div>
</div>

<script src="/assets/protocols-plus/assets/js/protocols.js"></script>

      <div id="react-footer-root" class="react-footer-root"><div id="footer" role="contentinfo" data-reactroot=""><div class="hor-line-top u-padding-s-top u-bg-white u-clr-grey7" style="border-color:#e9711c"></div><a class="anchor u-padding-s-left move-left els-footer-elsevier" href="https://www.elsevier.com/" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text"><svg viewBox="-3345 3440.027 140.01 24.333" style="width:104px;height:30px"><title>Elsevier</title><path id="E" style="fill:#E9711C" d="M-3343.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3344L-3343.999,3461.698"></path><path style="fill:#E9711C" d="M-3325.448,3461.698c2.074-0.118,2.83-0.502,2.83-2.832v-13.511c0-2.33-0.757-2.715-2.83-2.832v-0.591h8.884 v0.591c-2.243,0-3.027,0.472-3.027,2.891v13.009c0,1.652,0.056,2.625,1.71,2.625h4.008c3,0,4.4-2,5.576-4.1l0.673,0.118 l-1.71,5.222h-16.114V3461.698"></path><path style="fill:#E9711C" d="M-3307.122,3456.27h0.561c1.12,2.596,2.886,5.4,5.94,5.4c2,0,3.672-1.3,3.672-3.334 c0-1.652-1.626-3.1-4.176-4.927c-3.28-2.36-5.41-3.746-5.41-6.43c0-3.422,2.55-5.517,5.633-5.517c2.214,0,3,0.944,4.204,0.944 c0.476,0,0.56-0.266,0.476-0.737h0.561l0.645,6.076h-0.561c-0.785-2.625-2.523-5.19-5.354-5.19c-1.737,0-3.138,1.356-3.138,3.185 c0,1.918,1.933,3.157,5.016,5.016c2.523,1.504,5,3.362,5,6.254c0,3.245-2.608,5.752-5.97,5.752c-2.02,0-4.148-0.855-4.68-0.855 c-0.336,0-0.7,0.207-0.756,0.649h-0.56l-1.094-6.282"></path><path style="fill:#E9711C" d="M-3293.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3294L-3293.999,3461.698"></path><path style="fill:#E9711C" d="M-3265.839,3462.524h-0.42l-5.41-12.538c-0.896-2.065-1.71-4.16-2.83-6.136 c-0.478-0.826-1.346-1.327-2.3-1.327v-0.591h8.323v0.591c-0.785,0-2.354,0-2.354,1.15c0,0.384,0.87,2.45,1.653,4.308l4.063,9.676 l4.877-11.359c0.588-1.356,0.757-2.094,0.757-2.713c0-0.618-0.673-0.974-2.13-1.062v-0.59h5.941v0.591 c-0.337,0.06-0.7,0.117-1.037,0.295c-1.066,0.56-2.13,3.57-2.635,4.749l-6.5,14.957"></path><path style="fill:#E9711C" d="M-3255.472,3461.698c2.24,0,3.025-0.473,3.025-2.892v-13.393c0-2.42-0.784-2.89-3.025-2.891v-0.59h9.078v0.591 c-2.24,0-3.025,0.472-3.025,2.891v13.393c0,2.42,0.784,2.892,3.025,2.892v0.59h-9.08v-0.59"></path><path id="E_2_" style="fill:#E9711C" d="M-3244.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3245L-3244.999,3461.698"></path><path style="fill:#E9711C" d="M-3206,3461.698c-1.26-0.354-1.71-0.68-2.466-1.623l-6.166-7.609c3.027-0.65,5.13-2.185,5.13-5.547 c0-4.75-4.26-4.986-7.764-4.986h-9.191v0.591c2.24,0,3.026,0.472,3.026,2.891v13.393c0,2.42-0.785,2.892-3.026,2.892v0.59h9.08 v-0.59c-2.242,0-3.027-0.473-3.027-2.892v-5.604h2.551l7.314,9.086h4.54L-3206,3461.698 M-3220.399,3444.499 c0-1.387,0.337-1.476,2.186-1.476c2.774,0,5.3,0.974,5.3,4.308c0,3.6-2.887,4.63-5.914,4.631h-1.569v-7.463H-3220.399z"></path></svg></span></a><div class="panel-s u-bg-white u-padding-s-hor-from-md u-padding-xs-ver text-xs u-clear-both-from-xs u-clear-none-from-md"><ul class="u-margin-xs-bottom" style="list-style:none"><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/solutions/sciencedirect" id="els-footer-about-science-direct" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">About ScienceDirect</span></a><wbr/></li><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="/customer/authenticate/manra" id="els-footer-remote-access" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">Remote access</span></a><wbr/></li><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://science-direct-checkout.staging.ecommerce.elsevier.com/?" id="els-footer-shopping-cart" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">Shopping cart</span></a></li><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="http://elsmediakits.com" id="els-footer-advertise" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">Advertise</span></a><wbr/></li><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" id="els-footer-contact-support" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">Contact and support</span></a><wbr/></li><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" id="els-footer-terms-condition" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">Terms and conditions</span></a><wbr/></li><li class="u-display-inline"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/privacy-policy" id="els-footer-privacy-policy" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">Privacy policy</span></a></li></ul><p id="els-footer-cookie-message">We use cookies to help provide and enhance our service and tailor content and ads. By continuing you agree to the<!-- --> <a class="anchor u-margin-0-right" href="https://www.sciencedirect.com/legal/use-of-cookies" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text">use of cookies</span></a>.</p><p id="els-footer-copyright">Copyright © <!-- -->2019<!-- --> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</p></div><a class="anchor u-padding-l-bottom u-padding-s-hor u-position-relative move-bottom u-float-left-from-xs u-float-right-from-md move-right u-padding-0-hor u-margin-top-xs els-footer-relx" href="https://www.relx.com/" aria-label="RELX home page (opens in a new tab)" id="els-footer-relx-group" target="_blank" rel="nofollow" style="white-space:nowrap"><span class="anchor-text"><svg xmlns="http://www.w3.org/2000/svg" width="78" height="30" version="1" viewBox="0 0 280 65" alt="RELX group home page"><g transform="matrix(.13333 0 0 -.13333 0 65)"><path fill="#f4741e" d="M207 251c110 0 242 26 242 131 0 75-77 103-146 103C196 485 0 425 0 215 0 110 80 49 208 49c135 0 232 87 262 205C391 118 291 75 209 75 107 75 68 151 68 216c0 185 144 248 235 248 61 0 97-40 97-83 0-119-172-116-250-116h-38l-16-17c36-5 85-15 139-42C347 149 453 0 592 0c51 0 64 15 74 29-69-44-183 32-245 90-51 48-96 103-214 132"></path><path fill="#666666" d="M886 310c0 33-22 53-55 53h-77c-2 0-3-1-3-3v-99c0-2 1-3 3-3h77c33 0 55 20 55 52m7-261c-5 0-7 2-8 6l-69 145h-62c-2 0-3-1-3-4V55c0-4-2-6-6-6h-51c-4 0-6 2-6 6v361c0 3 2 5 6 5h139c66 0 115-45 115-111 0-49-27-86-69-102l76-152c2-4 0-7-4-7h-58M1010 416c0 3 3 5 6 5h218c4 0 6-2 6-5v-47c0-3-2-6-6-6h-158c-2 0-3-1-3-3v-91c0-2 1-3 3-3h127c4 0 6-2 6-6v-47c0-3-2-5-6-5h-127c-2 0-3-1-3-3v-95c0-2 1-3 3-3h158c4 0 6-2 6-5V55c0-4-2-6-6-6h-218c-3 0-6 2-6 6v361M1298 416c0 3 2 5 5 5h52c3 0 6-2 6-5V110c0-2 1-3 3-3h147c3 0 5-2 5-5V55c0-4-2-6-5-6h-208c-3 0-5 2-5 6v361M1775 49c-4 0-6 1-8 5l-73 127h-1l-73-127c-2-4-4-5-8-5h-58c-3 0-5 3-3 7l108 185-100 173c-2 4 0 7 3 7h58c4 0 6-2 8-6l65-112h1l65 112c2 4 5 6 9 6h57c4 0 5-3 3-7l-99-173 107-185c2-4 1-7-3-7h-58M1912 295l-2 1v104l-1 1h-32l-2 2v16l2 2h87l2-2v-16l-2-2h-32l-1-1V296l-2-1h-17M1988 419l2 2h16c1 0 2 0 3-2l33-78h1l33 78c1 2 2 2 3 2h16l2-2V296l-2-1h-16l-1 1v78h-1l-26-60-3-2h-11l-3 2-26 60h-1v-78l-2-1h-15l-2 1v123"></path></g></svg></span></a></div></div>
      <script id="store-json">
        var reduxData = {"config":{"topicSlug":"sanger-sequencing","domainSlug":"neuroscience","cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com","assetsBaseUrl":"/assets","baseEndpoint":"http://www.qa.sciencedirect.com","authData":{"redirect":"https://loadrc-id.elsevier.com/as/authorization.oauth2?platSite=SD%2Fscience&scope=openid+email+profile+els_auth_info+urn%3Acom%3Aelsevier%3Aidp%3Apolicy%3Aproduct%3Ainst_assoc&response_type=code&redirect_uri=https%3A%2F%2Fwww.qa.sciencedirect.com%2Fuser%2Fidentity%2Flanding&authType=SINGLE_SIGN_IN&prompt=none&client_id=SDFE-v6&state=retryCounter%3D0%26csrfToken%3D0b471224-ac80-4b9d-8b4d-de8a1c0c4f73%26idpPolicy%3Durn%253Acom%253Aelsevier%253Aidp%253Apolicy%253Aproduct%253Ainst_assoc%26returnUrl%3Dhttp%253A%252F%252Fwww.local.sdfe.sciencedirect.com%253A5107%252Ftopics%252Fneuroscience%252Fsanger-sequencing%252Fmethod%26prompt%3Dnone%26cid%3Dtpp-e3fc3bb6-8d74-45b7-9134-6c860b5c85f3","cookies":["csrf_token=0b471224-ac80-4b9d-8b4d-de8a1c0c4f73;Version=1;Domain=.sciencedirect.com;Path=/;HttpOnly","MIAMISESSION=3dc6331e-7d26-417f-b1d2-6a62ee999dd3:3743425902;Version=1;Domain=.sciencedirect.com;Path=/;HttpOnly","id_ab=B:100:3;Version=1;Domain=.sciencedirect.com;Path=/;Expires=Tue, 16 Aug 2039 16:31:42 GMT"],"isFallbackUser":true},"environment":"local","enableGlobalHeader":true,"enableGlobalHeaderSearch":true,"cdnBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com","supportHubId":25793,"requestPath":"/topics/neuroscience/sanger-sequencing/method","arsCDN":""},"topic":{},"pdfDownload":{},"renderTarget":{},"alerts":{"showAlertsModal":false},"entitlements":{}};
      </script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/10/js/babel-polyfill/6.26.0/babel-polyfill.min.js"></script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/26/js/react/16.8.4/react.production.min.js"></script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/26/js/react-dom/16.8.4/react-dom.production.min.js"></script>
      <script>
        window.assetsBaseURL = '/assets';
        window.pageData = {"page":{"businessUnit":"els:rp:st","language":"en","name":"foundationalcontent:topics:topicdetail","noTracking":"false","productName":"sd","type":"cp-cr","environment":"local","loadTimestamp":"1565973103102"},"content":[{"title":"neuroscience ^ sanger-sequencing","format":"MIME-XHTML","id":"sd:topic:6_200000463","type":"sd:topic"}],"visitor":{"accessType":"ae:ANON_GUEST","accountId":"","accountName":"","userId":"","ipAddress":"198.176.82.33","appSessionId":""}};
      </script>
      <script src="/assets/protocolsPlus.js" async></script>
      <script>
        function getPageLoadTime() {
          var returnValue = '';
          if (typeof(performance) !== 'undefined' && typeof(performance.timing) == 'object') {
              var timing = performance.timing;
              var startTime = timing.redirectStart || timing.fetchStart || timing.requestStart;
              var endTime = timing.domContentLoadedEventEnd || timing.domInteractive || timing.domComplete || timing.loadEventComplete;
              if (startTime && endTime && startTime < endTime) returnValue = endTime - startTime;
          }
          return returnValue.toString();
        }
        setTimeout(function() {
          window.pageData.page.loadTime = getPageLoadTime();
        }, 0);
        try { pageDataTracker.trackPageLoad(); }
        catch(e) { console.warn("There was an error loading or running Adobe DTM: ", e); }
      </script>
    </body>
  </html>